### PR TITLE
Updated OpenGrab EPM v3 information

### PR DESCRIPTION
**Pull request by Zentrus**
Added information regarding product status and updated links.

### CODE REVIEW

File: common/source/docs/common-electro-permanent-magnet-V3.rst [hunk 1]
```diff
@@ -4,6 +4,8 @@
 Electro Permanent Magnet Gripper (OpenGrab EPM v3)
 ==================================================
 
+⚠⚠ OpenGrab EPM v3 is no longer in production. Please use `Zubax EPM <https://zubax.com/epm>`__ as a replacement. ⚠⚠
```
Comment (Hwurzburg) [inline common/source/docs/common-electro-permanent-magnet-V3.rst:7]:
```suggestion
** OpenGrab EPM v3 is no longer in production. Please use `Zubax EPM <https://zubax.com/epm>`__ as a replacement. **
```

**Review by Hwurzburg — COMMENTED** [review]
symbol used does not appear well .....try just bold as suggested

File: common/source/docs/common-electro-permanent-magnet-V3.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-electro-permanent-magnet-V3.rst:7]: and needs a squash, I can do that for you when finalized

**Comment by Zentrus** [comment]
image
It doesn't bold the text. However, when the code is modified to remove the spaces, the link doesn't work
`**OpenGrab EPM v3 is no longer in production. Please use `Zubax EPM <https://zubax.com/epm>`__ as a replacement.**`
image

**Comment by Hwurzburg** [comment]
yes Sphinx does not like links inside bolded lines...I have fixed and squashed...will merge on CI pass

**Pull request merged**